services/core-web/src/components/noticeOfWork/applications/review/activities/SurfaceBulkSamples.js

```diff
@@ -52,10 +53,11 @@ export const SurfaceBulkSamples = (props) => {
       />
       <br />
       <br />
+      <h4>Processing Methods</h4>
       <Row gutter={16}>
         <Col md={12} sm={24}>
           <div className="field-title">
-            Processing Methods*
+            Describe handling and on-site processing methods
```

Comment:
Should this field still be required? (the "*" sign)

Reply:
no, the field isn't required on the form, I've updated the name and removed the required validation.